src/events/InteractionCreate.js

```
@@ -0,0 +1,86 @@
+      case 'help':
+        return await helpHandler(i);
+      case 'refresh':
+        return await refreshHandler(i);
+      case 'repost':
+        return await repostHandler(i);
+      case 'weeklyschedule':
+        return await weeklyScheduleHandler.handleWeeklySchedule(i);
+      case 'generate':
+        return await generateHandler(i);
+      case 'post':
+        return await postHandler(i);;
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)